src/Simulation.jl

```diff
@@ -271,8 +271,12 @@ function rops(ssys::SystemSimulation, t)
     vdiffs = Array{Any,1}(undef, length(domains))
     vCvave = Array{Any,1}(undef, length(domains))
     vphi = Array{Any,1}(undef, length(domains))
-    ropmat = spzeros(Nrxns, Nspcs)
-    start = 1
+    if any([domain isa FragmentBasedConstantTrhoDomain for domain in domains])
```

Comment:
What if you have multiple domains that track mass?

Are we using Nrxns/Nspcs to handle indexing elsewhere?....more generally are the indices all shifted properly everywhere and the addition of the mass column from any arbitrary domain isn't going to cause us to extract the wrong species rop information?...do we need a property/function of the SystemSimulation that provides the right number of columns?

Reply:
We're not currently supporting multiple domains that track mass. I can't think of why one would want to do that, and I don't think we should support a hypothetical possibility without an actual use case. 

I don't think we are using Nrxns/Nspcs to handle indexing elsewhere. Changing this will also only affect simulation that uses the `FragmentBasedConstantTrhoDomain`, which I wrote special functions for its rop. It shouldn't affect other domains' rop.

Reply:
Based on discussion should keep the rop matrix a consistent shape. Two good options: 
1) Split the mass rop computation into a separate function 
2) Have a new function that isn't rops! that computes both the rops and a separate mass rops vector

Reply:
Thanks for the suggestion! Option 1 is inefficient as the `massropvec` computation requires information already evaluated during the computation of `ropmat`. I will change the code based on Option 2.